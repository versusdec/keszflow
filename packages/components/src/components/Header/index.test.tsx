import React from 'react'
import { Header } from './index'

import { mount, shallow } from 'enzyme'

describe('Header', () => {
  const getComponent = () => <Header />
  it('Should render correctly with Shallow', () => {
    const component = shallow(getComponent())
    expect(component).toMatchSnapshot()
  })
  it('Should render correctly with mount', () => {
    const component = mount(getComponent())
    expect(component).toMatchSnapshot()
  })
})
