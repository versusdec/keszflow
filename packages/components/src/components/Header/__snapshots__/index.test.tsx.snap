// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Should render correctly with Shallow 1`] = `ShallowWrapper {}`;

exports[`Header Should render correctly with mount 1`] = `ReactWrapper {}`;
